### ISSUE TITLE
@font-face with a curly brace on the next line is reported as a syntax error

### ISSUE THREAD

**Issue by phistuck** [gmetais/YellowLabTools#45]
If your CSS includes a font declaration with the curly braces on a new line, like this -
```
@font-face
{
 font-family: 'foo';
 src: url('foo.woff') format('woff');
}
```

The report says there is a syntax error in the CSS -
```
CSS syntax error

F
0/100
Value: 1

Yellow Lab Tools failed to parse a CSS file. I doubt the problem comes from the css parser.

Maybe a CSS validator can help you.

THE-URL (Error: CSS parsing failed: @font-face missing '{' @ 7:11)
```

**Comment by gmetais** [comment]
Hi @phistuck!

Someone told me the same thing, and i opened an issue on the css parsing module:
https://github.com/reworkcss/css/issues/61

I can't tell when it will be resolved, but thanks a lot for reporting the issue.

**Comment by phistuck** [comment]
Thank you for the prompt response. :)

**Comment by phistuck** [comment]
When I try https://github.com/reworkcss/css/blob/master/lib/parse/index.js and use -
```
module.exports("@font-face\r\n{\r\n font-family: 'Ar';\r\n src: url('p.woff') format('woff');\r\n}")
```
(Well, I actually removed `module.` to make it work almost out of the box)
It parses successfully. It might have been fixed already and you just have not pulled it in yet.


Yep. A month ago...
https://github.com/reworkcss/css/commit/46d2b22aa43697ae78bd829f2770895b7da6883e

**Comment by gmetais** [comment]
First, sorry I didn't realize you were talking about `@font-face` and I was talking about `@keyframes`.

Then, thank you for your pull request on https://github.com/reworkcss/css about @font-faces :)

Now I'll work on this issue https://github.com/reworkcss/css/issues/59 and hope a v2.2.0 is released!

**Comment by phistuck** [comment]
You can just backport the changes yourself, right?

**Comment by gmetais** [comment]
Yes, @keyframes haha, I still confuse them :)

You mean I copy-paste the changes in my repository? It's technically possible but I can't. It's not the proper way to do so.

Also, YLT doesn't directly rely on reworkcss/css, it depends on macbre/analyze-css that depends on reworkcss/css. If I make the change on YLT, the problem wouldn't be solved on macbre's project, and that's not what I want.

**Comment by phistuck** [comment]
I meant that you can do it on a release branch only or something. Your tools is not useful if it stops parsing my CSS right at its beginning.
Remember that there is nothing you can do about macbre or reworkcss/css, since the author simply has to pull/merge the changes and/ore release a version.

It is fine to have local patches while waiting for the other repository to release, this is how the world works, regarding security issues, for example.

But I understand you approach. I just think it is a shame, because your tool is not that useful with my website.

**Comment by gmetais** [comment]
This was fixed in 1.3.0 via #57.
I forgot to close the issue!
